### ISSUE TITLE
Deleting cache?

### ISSUE THREAD

**Issue by nicopap** [coursier/coursier#1911]
Hello, I found coursier's cache takes 3GB on my system and I want to delete it (the cache). However, there is no "official" ways of doing it (through cli command flags or detailed in the documentation in https://get-coursier.io). And searching the issues in this repo, there is no mention of deleting cache.

With a regular package manager (like `pacman`) it would be dangerous to delete files owned by the app recklessly, so I'm assuming the safest thing and thinking it might be the same for coursier.

I don't want to break my setup so let's be clear:
The cache is located at `C:\Users\User\AppData\Local\Coursier` on my system. What's the safest way to delete coursier cache?

* Deleting the `cache` directory in `Coursier`?
* deleting `v1` directory inside `cache`?
* Some other process (like an undocumented `coursier` cli flag)

**Comment by mikela** [comment]
You can safely delete the `cache` folder, it will be generated again.

**Comment by alexarchambault** [comment]
Yes, the cache folder is re-filled automatically when it's deleted.

@nicopap Note that the cache of coursier doesn't have the same purpose as the one of pacman or dpkg, for example. For those tools, the data in the cache are only used temporarily, while packages are being installed. In the case of coursier, these files are meant to be used as is, directly from the cache. For example, when you compile a project from sbt, JAR files living in the cache are used during compilation, or when running projects. When installing apps with `cs install`, the installed apps directly use JARs from the cache (and just download them if they are missing).

**Comment by nicopap** [comment]
Neat. Thank for the answer. It seems this sums it up. I hope the issue can be a reference for other people with the same questions. Is it worth keeping the issue open as a "reminder" to add more docs concerning cache deletion and location? Or we are good and it can be closed safely?

**Comment by som-snytt** [comment]
To answer the rhetorical question, it would be nice to have it documented and also to have a tool command to clean cruft, for some definition of cruft. Perhaps a command to define a checkpoint or restore state, and then a clean command that restores it by removing what doesn't belong (as more convenient than setup).

**Comment by adorbabble** [comment]
I am a little confused. If I’d like to only keep the latest versions of packages (I am just learning Scala for now), what is the easiest way to proceed ? When I want to update, should I delete the contents of the `cache` folder and use `cs setup` again (or maybe `cs install` ?) instead of `cs update` ? How does `cs` know what is installed? Does Coursier need to scan the `cache` folder every time ? I haven’t seen anything like a file where Coursier keeps tracks of what (and which versions) is installed
